suite-native/device/src/hooks/usePinAction.tsx

```diff
@@ -63,7 +65,7 @@ export const usePinAction = ({ type, onSuccess }: PinActionProps) => {
         [showToast],
     );
 
-    const showError = useCallback(
+    const showFallbackError = useCallback(
```

Comment:
Naming is confusing for me, would vote for `showErrorFallback` or `showErrorDefault`.